.github/workflows/tests.yml

```
@@ -40,10 +42,29 @@ jobs:
         if: github.event_name == 'pull_request'
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ github.token }} # required
-
+      
+      - name: Failures Summary and Bottle Result
+        if: always()
+        uses: Homebrew/actions/failures-summary-and-bottle-result@master
+        with:
+          workdir: ${{ github.workspace }}
+          result_path: bottles/steps_output.txt
+          step_name: 'Build failure summary on ${{ matrix.os }}'
+          collapse: 'true'
+  
       - name: Upload bottles as artifact
         if: always() && github.event_name == 'pull_request'
         uses: actions/upload-artifact@main
         with:
           name: bottles
           path: '*.bottle.*'
+
+      - name: Post comment 
+        uses: Homebrew/actions/post-comment@master
+        with:
+          token: ${{ github.token }}
+          issue: ${{ github.event.number }}
+          body: "@${{github.actor}} bottle request for ${{ runner.os }} (${{github.event.repository.html_url}}/actions/runs/${{github.run_id}})."
+          bot_body: "Bottle request for ${{ runner.os }} (${{github.event.repository.html_url}}/actions/runs/${{github.run_id}})."
+          bot: BrewTestBot
+  
```

These are a little bit useful, but I still don't know how to let the CI fail